### ISSUE TITLE
Block update profiling

### ISSUE THREAD

**Issue by LunNova** [MinimallyCorrect/TickProfiler#44]
We should keep track of the number of block update and TE updates being sent out.

High numbers will cause higher CPU load on both client (resulting in lower FPS) and server, and also increased number of packets being sent.

**Comment by rmnaderdev** [comment]
Yes! This would be very nice. I think it is missing info that the old /profile e (everything) used to tell us. I have an issue on one of my servers right now and I can't see what block/TE is giving me trouble.
